source/features/one-click-review-submission.tsx

```diff
@@ -58,50 +64,37 @@ function addButtons(radios: HTMLInputElement[]): void {
 			button.prepend(<FileDiffIcon className="color-fg-danger"/>);
 		}
 
-		container.append(button);
+		actionsRow.append(button);
 	}
 
 	// Remove original fields at last to avoid leaving a broken form
 	for (const radio of radios) {
 		radio.closest('.form-checkbox')!.remove();
 	}
 
-	// The selector excludes the "Cancel" button
-	select('[type="submit"]:not([name])', container)!.remove();
-}
-
-function handleSubmission(event: DelegateEvent): void {
-	// Delay disabling the fields to let them be submitted first
-	setTimeout(() => {
-		for (const control of select.all('button, textarea', event.delegateTarget)) {
-			control.disabled = true;
-		}
-	});
+	originalSubmitButton.remove();
 }
 
-function init(signal: AbortSignal): false | void {
-	// Freeze form to avoid duplicate submissions
-	delegate('[action$="/reviews"]', 'submit', handleSubmission, {signal});
-
-	// This will prevent submission when clicking "Comment" and "Request changes" without entering a comment and no other review comments are pending
-	delegate('[action$="/reviews"] button', 'click', ({delegateTarget: {value, form}}) => {
-		const pendingComments = looseParseInt(select('.js-reviews-toggle .js-pending-review-comment-count'));
-		const submissionRequiresComment = pendingComments === 0 && (value === 'reject' || value === 'comment');
-		select('#pull_request_review_body', form!)!.toggleAttribute('required', submissionRequiresComment);
-	}, {signal});
```

Comment:
Dropped. This can fail once the `.js-reviews-toggle .js-pending-review-comment-count` is not found the the number of comments _loosely_ becomes 0, blocking submission.